### PR TITLE
Update pack versions

### PR DESCRIPTION
**Pull request by ritawatson**
This PR updates the versions of packs listed below - please check the docs for packs you updated:

**AWS EFS CSI** - This doc is also refactored to use the new Packs Doc Layout. Please see questions in the Troubleshooting section. It seems the existing information is more usage than Troubleshooting - should we move it to the Usage section? I added new Troubleshooting information for your review.

Kong
Azure Disk CSI
K8s Dashboard
External-DNS
Open Policy Agent
Calico CNI
External Secret Operator

For docs purposes, this PR also adds a shortcut to the Makefile.

**[Preview](https://deploy-preview-1093--docs-spectrocloud.netlify.app/integrations)**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **zulfilee**
You can assign the PR to them by writing `/assign @zulfilee` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**
- **[prow/OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/prow/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-aws-efs.md [hunk 1]
```diff
@@ -22,25 +22,60 @@ Amazon Elastic File System (Amazon EFS) is a scalable file storage that allows f
...
+| gidRangeStart    |                  | 1000                  | Starting range of the Portable Operating System Interface(POSIX) group Id to be applied for access point root directory creation (optional).                                                                                      |
+| gidRangeEnd      |                     | 2000                  | End range of the POSIX group Id (optional).                                                                                                                                                                                                 |
+| basePath         |                  | /base_efs             | Path under which access points for dynamic provisioning is created. <br /> If this parameter is not specified, access points are created under the root directory of the file system.                                             |
+
+
+## Usage
+
+There are two ways to add AWS EFS to Palette:
+
+- Add EFS as a CSI layer in AWS/EKS.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:72]:
```suggestion
- Add EFS as a base CSI layer in a cluster profile.
```

File: content/docs/06-integrations/00-aws-efs.md [hunk 2]
```diff
@@ -114,18 +154,139 @@ While adding the AWS EFS layer, the following parameters can be configured:
...
+
+
+- Add EFS as an Add-on layer, which will create a new storage class using the AWS EFS file system.
+
+
+### Policy Information
+
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:168]: Why are we showing this IAM policy? We should add an explanation.

File: content/docs/06-integrations/00-aws-efs.md [hunk 3]
```diff
@@ -114,18 +154,139 @@ While adding the AWS EFS layer, the following parameters can be configured:
...
+  ]
+}
+```
+
+</Tabs.TabPane>
+</Tabs>
+
+
+# Troubleshooting 
+**QUESTION: Is information below about "Storage Class" and "PersistentVolumnClaim" really information for the Usage section?**
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:211]: Yes, this is usage information.

File: content/docs/06-integrations/00-aws-efs.md [hunk 4]
```diff
@@ -114,18 +154,139 @@ While adding the AWS EFS layer, the following parameters can be configured:
...
+
+</Tabs.TabPane>
+</Tabs>
+
+
+# Troubleshooting 
+**QUESTION: Is information below about "Storage Class" and "PersistentVolumnClaim" really information for the Usage section?** 
+
+**Should the Troubleshooting section list the following points instead?**
+**Is is safe to include the commands shown or is there the possibility they could change?. If so, the doc can list the bullet points and refer users to the Amazon troubleshooting site for more details.**
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:214]: The commands can be used by the section needs clarification and clean-up.

File: content/docs/06-integrations/00-aws-efs.md [hunk 5]
```diff
@@ -114,18 +154,139 @@ While adding the AWS EFS layer, the following parameters can be configured:
...
+  spec:
+    mountOptions:
+      - iam
+  ```
+- **IAM role:** Verify the Amazon EFS CSI driver controller service account associates with the correct IAM role and the IAM role has the required permissions.
+
+  Run the following command:
+
+  ```bash
+  kubectl describe sa efs-csi-controller-sa -n kube-system
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:258]:
```suggestion
  kubectl describe sa efs-csi-controller-sa --namespace kube-system
```
We should always show the long-form of command to help the reader.

File: content/docs/06-integrations/00-aws-efs.md [hunk 6]
```diff
@@ -114,18 +154,139 @@ While adding the AWS EFS layer, the following parameters can be configured:
...
+  
+  You shoud see this annotation:
+
+  ```bash
+  eks\.amazonaws\.com/role-arn"="arn:aws:iam::111122223333:role/AmazonEKS_EFS_CSI_Driver_Policy
+  ```
+- **Driver Pods:** Verify the EFS CSI driver pods are running. Issue the following command to display a list of controller pods and node pods running in your cluster:
+
+  ```bash
+  kubectl get all -l app.kubernetes.io/name=aws-efs-csi-driver -n kube-system
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:269]:
```suggestion
  kubectl get all --label app.kubernetes.io/name=aws-efs-csi-driver --namespace kube-system
```

File: content/docs/06-integrations/00-aws-efs.md [hunk 7]
```diff
@@ -114,18 +154,139 @@ While adding the AWS EFS layer, the following parameters can be configured:
...
+  ```
+
+- **File system won't mount:** Verify the EFS mount operation from the EC2 worker node where the pod is failing to mount the file system. Log in to the Amazon EKS worker node where the pod is scheduled. Then, use the EFS mount helper to try to manually mount the EFS file system to the worker node. You can run the following command to test:
+
+  ```bash
+  sudo mount -t -efs -o tls file-system-dns-name efs-mount-point/
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:275]:
```suggestion
  sudo mount --types -efs --options tls file-system-dns-name efs-mount-point/
```

File: content/docs/06-integrations/00-aws-efs.md [hunk 8]
```diff
@@ -163,24 +324,35 @@ Events:
...
+data "spectrocloud_registry" "container_registry" {
+  name = "Public Repo"
+}
 
+data "spectrocloud_pack_simple" "csi-aws-efs" {
+  name    = "csi-aws-efs"
+  version = "1.4.0"
+  type = "helm"
+  registry_uid = data.spectrocloud_registry.container_registry.id
+}
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:349]:
```suggestion
data "spectrocloud_registry" "public_registry" {
  name = "Public Repo"
}

data "spectrocloud_pack_simple" "csi-aws-efs" {
  name    = "csi-aws-efs"
  version = "1.4.0"
  type = "helm"
  registry_uid = data.spectrocloud_registry.public_registry.id
}
```

Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-aws-efs.md:349]: @ritawatson, so the current Terraform code is if you are using this Pack as an add-on to a cluster profile.

When using this Pack as a base layer, you need the following terraform code.

```
data "spectrocloud_registry" "public_registry" {
  name = "Public Repo"
}
data "spectrocloud_pack_simple" "aws-efs" {
  name    = "aws-efs"
  version = "1.4.0"
  type = "helm"
  registry_uid = data.spectrocloud_registry.public_registry.id
}
```

Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-aws-efs.md:349]: Pink is system app (storage layer), orange is addon
CleanShot 2023-01-30 at 19 35 56@2x

Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-aws-efs.md:349]: I asked in a thread that I tagged you on if this is the plan.

File: content/docs/06-integrations/00-kong.md [hunk 9]
```diff
@@ -9,20 +9,38 @@ logoUrl: 'https://registry.spectrocloud.com/v1/kong/blobs/sha256:600f20583f85cca
...
 import WarningBox from 'shared/components/WarningBox';
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
 
 # Kong Ingress Controller
 
-Kong integration is an [Ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers) responsible for fulfilling the Ingress, usually with a load balancer, though it may also configure your edge router or additional frontends to help handle the traffic.
+The Kong integration is an Ingress Controller for Kubernetes that configures ingress with a load balancer. Kong can also configure your edge router or additional frontends to help handle incoming requests.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kong.md:19]:
```suggestion
The Kong integration is an Ingress Controller for Kubernetes that configures ingress with a load balancer. You can use the Kong as an application load balancer for your application.
```
The later statement is confusing so let's just remove it.

File: content/docs/06-integrations/00-kong.md [hunk 10]
```diff
@@ -9,20 +9,38 @@ logoUrl: 'https://registry.spectrocloud.com/v1/kong/blobs/sha256:600f20583f85cca
...
+* **1.4.0**
+
+</Tabs.TabPane>
+
+</Tabs>
 
 ## Components
 
-The integration will add Kong Ingress Controller which will expose a service of type LoadBalancer
+The integration adds the Kong Ingress Controller, which exposes a service of type LoadBalancer.
```
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kong.md:41]:
```suggestion
The integration adds the Kong Ingress Controller, which exposes a Kubernetes service of type LoadBalancer.
```
We use service a bit through our documentation so being specific helps a bit.

**Review by karl-cardenas-coding — COMMENTED** [review]
Overall good, left some feedback about the EFS pack.

File: content/docs/06-integrations/00-longhorn.md [hunk 11]
```diff
@@ -22,7 +22,7 @@ Longhorn creates a dedicated storage controller for each block device volume and
 
 # Prerequisites
 
-- Kubernetes cluster is at least version 1.22 and not higher than 1.24.
+- Kubernetes cluster is 1.25 or higher.
```
Comment (brathina-spectro) [inline content/docs/06-integrations/00-longhorn.md:25]: Kubernetes version for longhorn should be v1.22 at a minimum. 
The first version of longhorn v1.3.1 is not supported for k8s versions 1.25 and higher.
So, we're rolling out a new version of longhorn v1.4.0. 

This is an example where external dependencies (in this case Kubernetes) vary for a specific version of the pack and the pre-req / limitations should be documented for every version.

**Review by ritawatson — COMMENTED** [review]
Incorporated review feedback.

File: content/docs/06-integrations/00-aws-efs.md [hunk 12]
```diff
@@ -114,18 +154,139 @@ While adding the AWS EFS layer, the following parameters can be configured:
...
+    mountOptions:
+      - iam
+  ```
+- **IAM role:** Verify the Amazon EFS CSI driver controller service account associates with the correct IAM role and the IAM role has the required permissions.
+
+  Run the following command:
+
+  ```bash
+  kubectl describe sa efs-csi-controller-sa -n kube-system
+  ```
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/06-integrations/00-aws-efs.md:346]: fixed typo.

File: content/docs/06-integrations/00-aws-efs.md [hunk 13]
```diff
@@ -22,25 +22,60 @@ Amazon Elastic File System (Amazon EFS) is a scalable file storage that allows f
 
 Palette handles setting up the AWS EFS as a volume with ease when adding the PersistentVolume storage container. Palette will dynamically provision the AWS EFS storage layer for the worker node.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/06-integrations/00-aws-efs.md:23]: Removed.

File: content/docs/06-integrations/00-aws-efs.md [hunk 14]
```diff
@@ -22,25 +22,60 @@ Amazon Elastic File System (Amazon EFS) is a scalable file storage that allows f
...
+* **1.4.9**
+* **1.4.0**
 
-# Prerequisites
+## Prerequisites
 
 - Create the Identity and Access Management (IAM) role that allows the driver to manage AWS EFS access points. See the [Introducing Amazon EFS CSI dynamic provisioning](https://aws.amazon.com/blogs/containers/introducing-efs-csi-dynamic-provisioning/) blog for information on `EFSCSIControllerIAMPolicy`.
 
 
 - Have a filesystem created and available before you provision AWS EFS to Palette.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/06-integrations/00-aws-efs.md]: Incorporated.

**Comment by ritawatson** [comment]
@brathina-spectro @karl-cardenas-coding This PR is updated with suggested changes. Please take a look and approve if all looks good. 

I refactored Longhorn to use the new Packs layout, as the prerequisite is different between the two versions.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#pullrequestreview-1281965334" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by brathina-spectro — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#pullrequestreview-1283562316" title="Approved">brathina-spectro</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1093#pullrequestreview-1281965334" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**